src/biotite/structure/atoms.py

```diff
@@ -909,7 +932,17 @@ def __init__(self, depth, length):
             self._coord = None
         else:
             self._coord = np.full((depth, length, 3), np.nan, dtype=np.float32)
-    
+
+    def __repr__(self):
+        """Represent AtomArray as a string for debugging."""
+        arrays = ''
+        for i in range(0, self.stack_depth()):
+            if len(arrays) == 0:
+                arrays = self.get_array(i).__repr__()
+            else:
+                arrays = arrays + ',\n' + self.get_array(i).__repr__()
+        return f'stack([{arrays}])'
```

Comment:
Similar to the comment above, i would like to request another layer of indentation and an empty line between each model to visibly separate them.